README.md

```diff
@@ -1,693 +1,47 @@
-  claimLink: "claim-name"
-  iamUserSecret: accountName-secret
-```
-
-*awsAccountID* is updated with the account ID of the aws account that is created by the account controller
-
-*claimLink* holds the name of the accountClaim that has claimed this account CR
-
-*iamUserSecret* holds the name of the secret containing IAM user credentials for the AWS account
-
-#### 3.2.4. Status
-
-Updates the Account CR
-
-```yaml
-status:
-  claimed: false
-  conditions:
-  - lastProbeTime: 2019-07-18T22:04:38Z
-    lastTransitioNTime: 2019-07-18T22:04:38Z
-    message: Attempting to create account
-    reason: Creating
-    status: "True"
-    type: Creating
-  rotateCredentials: false
-  state: Failed
-  supportCaseID: "00000000"
-```
-
-*state* can be any f the account states defined in the constants below:
-
-- AccountPending indicates an account is pending
-- AccountCreating indicates an account is being created
-- AccountFailed indicates account creation has failed
-- AccountReady indicates account creation is ready
-- AccountPendingVerification indicates verification (of AWS limits and Enterprise Support) is pending
-
-*claimed* is true if `currentAcctInstance.Status.State == AccountReady && currentAcctInstance.Spec.ClaimLink != "`
-*rotateCredentials* updated by the secretwatcher pkg which will set the bool to true triggering an reconcile of this controller to rotate the STS credentials
-*supportCaseID* is the ID of the aws support case to increase limits
-*conditions* indicates the last state the account had and supporting details
-
-#### 3.2.5. Metrics
-
-Update in the account-controller
-
-```txt
-MetricTotalAWSAccounts
-```
-
-### 3.3. AccountClaim Controller
-
-The accountClaim-controller is triggered when an accountClaim is created in any namespace. It is responsible for following behaviours:
-
-1. Sets account `spec.ClaimLink` to the name of the accountClaim
-2. Sets accountClaim `spec.AccountLink` to the name of an unclaimed Account
-3. Creates a secret in the accountClaim namespace that contains the credentials tied to the aws account in the accountCR
-4. Sets accountClaim `status.State = "Ready"`
-
-#### 3.3.1. Constants and Globals
-
-```go
-AccountClaimed          = "AccountClaimed"
-AccountUnclaimed        = "AccountUnclaimed"
-awsCredsUserName        = "aws_user_name"
-awsCredsAccessKeyID     = "aws_access_key_id"
-awsCredsSecretAccessKey = "aws_secret_access_key"
-```
+## Quick Start
 
-#### 3.3.2. Spec
+This Quick Start assumes that you are working on a team that already has AWS Accounts set up for development/testing.  For first time setup, see the prerequisites documentation page.
 
-Updates the accountClaim CR
+First, set up your required environment variables:
 
-```yaml
-spec:
-  accountLink: osd-{accountName}
-  aws:
-    regions:
-    - name: us-east-1
-  awsCredentialSecret:
-    name: aws
-    namespace: {NameSpace}
-  legalEntity:
-    id: 00000000000000
-    name: {Legal Entity Name}
+```bash
+export AWS_PAGER=
```

Comment:
Huh I've never heard of this. What does this do for us?

Reply:
Ahh, this is an aws cli 2.0 thing.  I have this set because when I run `make test-all` it will stop at any time there's an AWS call and put it into `less` by default requiring me to interact to keep the tests moving.  See also: https://docs.aws.amazon.com/cli/latest/userguide/cli-usage-pagination.html#cli-usage-pagination-clientside

Reply:
Coolio. Nice to have